app/poro/household.rb

```diff
@@ -0,0 +1,11 @@
+class Household
```

Comment:
Good work.  I think the file naming convention is plural, app/poros/household.rb

Reply:
oh yes, Kevin is right. It is plural!